### ISSUE TITLE
Add benchmarks for using basic nfts

### ISSUE THREAD

**Issue by ethanfrey** [iov-one/weave#224]
While we have some [basic application-level benchmarks](https://github.com/iov-one/weave/blob/master/cmd/bcpd/app/app_test.go#L595-L767) for sending tokens, as well as sending tokens from multisig contracts, we have no benchmarks for the use of NFTs. This information would be very useful to provide a baseline before we undertake a larger refactoring and change of NFT implementation.

After some discussion, it seems that package-level benchmarks, directly calling the handler and querying the bucket, would be most effective. To make them realistic, we should benchmark various "block sizes" meaning to create a leveldb-backed store and wrap it with a cache (like BeginBlock), and call the Handler N times (with different calls) before committing it. We can directly set the authorized signatures in the Context, as is done in many unit tests, just run a larger number of them as a benchmark.

This story involves benchmarking the creation and update of the following NFTs:

Update: just focus on x/nft/username

* [ ] `x/nft/blockchain`
* [x] `x/nft/ticker` (https://github.com/iov-one/weave/pull/270)
* [ ] `x/nft/username`
* [ ] `x/nft/bootstrap_node`

Some of them obviously require other tickers to be created first, so make any setup phase necessary. Two parameters to adjust when running the suite for any given ticker are: 
* number of transactions/messages per block (where the commit/cache logic can be averaged out)
* number of tokens existing before the benchmark (eg. preload 10 or 1000 tickers before attaching tokens to the username)

Please test both the creation of new NFTs as well as the main update methods (add/remove chainaddress to username, anything else?)

Benchmark must be done separately for every message. For example a benchmark for `username.RemoveChainAddressMsg` must test only processing this message and not the whole flow of adding and address and removing it. Adding and address is part of the setup process and must not be measured in this benchmark.

Handing queries is now split into a separate story.

**Comment by ethanfrey** [comment]
Update. After #282 only the username nft is priority to be benchmarked. Other benchmarks can be done much later (when we start on namespace implementation)

**Comment by ethanfrey** [comment]
We removed 90% of this, and I don't know if benchmarks are needed
